src/main/java/core/config/users/User.java

```diff
@@ -17,6 +19,19 @@
     private String password;
     private String alias;
 
+    /**
+     * User constructor.
+     *
+     * @param username represents the user's username
+     * @param password represents the user's password
+     * @param alias    represents the user's alias

```

Comment:
Extra space between alias    and represents

Reply:
Solved.